Project-Aurora/Project-Aurora/Utils/EnumUtils.cs

```diff
@@ -77,7 +77,43 @@ public object Convert(object value, Type targetType, object parameter, CultureIn
 
             if (value == null || string.IsNullOrEmpty(value.ToString()) || (value.GetType() != EnumType && value.GetType() != typeof(string)))
                 return DefaultEnum.GetDescription();
-            return (this.StringToEnum(EnumType, value.ToString())).GetDescription();
+
+
+            if (EnumType == typeof(PreferredKeyboard))
```

Comment:
Why not make the descriptions of the layouts more descriptive instead of doing this? This really seems like code smell

Reply:
It is support a lot of OEMS, so it is going to change by OEM name.

Reply:
Yeah but doing that in this EnumUtils class leads to code mess, can you just change the descriptions to include OEMs names?